annotated_test.go

```diff
@@ -1474,6 +1474,134 @@ func TestAnnotate(t *testing.T) {
 	})
 }
 
+func TestAnnotateApplyFail(t *testing.T) {
+	t.Parallel()
+
+	type a struct{}
+	type b struct{ a *a }
+	newA := func() *a { return &a{} }
+	newB := func(a *a) *b {
+		return &b{a}
+	}
+
+	var (
+		errTagSyntaxSpace            = ` multiple tags are not separated by space`
```

Comment:
Could we re-use the error declarations in annotated .go, but call it's errors.Error() value in the test cases?

Reply:
annoted_test is under fx_test, no need to export the error.